state/cosmosdb/cosmosdb.go

```diff
@@ -16,8 +16,8 @@ import (
 	"github.com/a8m/documentdb"
 )
 
-// StateStore is a CosmosDB state store
-type StateStore struct {
+// CosmosDB is a CosmosDB state store
+type CosmosDB struct {
```

Comment:
package name is cosmosdb and you renamed StateStore to CosmosDB. cosmosdb.CosmosDB doesn't look good to me. Do you have the better idea for this ? maybe StateStore would be better.. cc/ @yaron2

Reply:
Oh i take a look on other implementation (eg etcd), and would like to make it consistent. Will be easier for consumer

Reply:
@shalabhms @youngbupark noted and thanks guys, let me update the PR.